jax/interpreters/xla.py

```diff
@@ -56,6 +56,148 @@ def _map(f, *xs): return tuple(map(f, *xs))
 def identity(x): return x
 
 
+### lazy sub-language
```

Comment:
Hmm what module do you suggest? `jax.util.lazy`? It's a bit xla.py-specific because of `stage_lazy_expr` using the XLA computation builder, and it's only consumed here.

Reply:
I moved it to its own module!